psh/sysexec/sysexec.c

```diff
@@ -34,7 +37,7 @@ static void psh_sysexecinfo(void)
 
 static void psh_sysexecUsage(void)
 {
-	fputs("Usage: sysexec [-m datamap] [-M codemap] progname [args]...\n", stderr);
+	fputs("Usage: sysexec [-d] [-i] [-m datamap] [-M codemap] progname [args]...\n", stderr);
```

Comment:
I think it'd be nice to add short descriptions of the options (as `-i` is not self-explanatory)